### DISCUSSION TITLE
Self-nomination to Contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#155] [Stacks Team]
Hi @paketo-buildpacks/stacks-maintainers

I'd like to nominate myself for the Contributor role. I've been working on the jammy tiny stack and will continue to contribute to stacks for the foreseeable future.

👍 / 👎 

Thanks,
Rob

**Comment by robdimsdale** [comment]
@paketo-buildpacks/steering-committee looks like we have a majority of stacks maintainers giving a 👍 on me becoming a stacks contributor

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on becoming a Stacks Contributor!! 🎉
